dcpy/test/conftest.py

```diff
@@ -0,0 +1,34 @@
+import boto3
+import os
+import pytest
+
+from moto import mock_s3
+from dcpy.utils.s3 import client
+
+TEST_BUCKET_NAME = "test-bucket"
+TEST_DIR_NAME_1 = "test-dir-1"
+TEST_DIR_NAME_2 = "test-dir-2"
+TEST_FILE_NAME = "test-file"
```

Comment:
Can we move these into the test file itself? Or do they need to be in the fixtures?

Reply:
@alexrichey - if it's okay, I would like to revisit keeping or moving these constants after writing more tests. I may need these variables across multiple test files.